### DISCUSSION TITLE
Unable to launch wayvnc from KDE Plama

### DISCUSSION THREAD

**Discussion by atsen-dev** [any1/wayvnc#246] [General]
Running on Manjaro KDE ARM on Wayland obviously, I've installed wayvnc 0.5.0 and when I launch `wayvnc` or `wayvnc 0.0.0.0` I have this message and nothing more : 

wayvnc: symbol lookup error: wayvnc: undefined symbol: nvnc_set_cut_text_receive_fn

I precize that I've aml (0.3.0) and neatvnc (0.6.0) installed. I don´t know if i'm missing some runtime dependencies, I'm unable to run waynvnc.

**Comment by any1** [comment]
This looks like a problem with the way that wayvnc has been distributed/built/packaged.

Also, wayvnc is currently not compatible with KDE.
